retroshare-gui/src/gui/TheWire/PulseViewGroup.ui

```diff
@@ -538,10 +558,25 @@
        </widget>
       </item>
      </layout>
+     <zorder>widget_header</zorder>
+     <zorder>widget_publish</zorder>
+     <zorder>line_1</zorder>
+     <zorder>widget_replies</zorder>
+     <zorder>line_2</zorder>
+     <zorder>widget_actions</zorder>
+     <zorder>frame_masthead</zorder>
     </widget>
    </item>
   </layout>
  </widget>
+  <customwidgets>
+  <customwidget>
+   <class>CustomFrame</class>
+   <extends>QFrame</extends>
+   <header>D:\new\RetroShare\retroshare-gui\src\gui\TheWire\CustomFrame.h</header>

```

Comment:
This is concerning: normally you should never put an absolute file path in a custom widget definition, because people on other computers will not be able to compile.